src/Parade.test.js

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react';
+import Parade from './Parade';
+
+test('the emoji that renders depends on the animal prop', () => {
+
+  render(<Parade animalArray={['fox', 'raccoon', 'rat', 'orangutan']} />);
+  const foxEmoji = screen.getByText(/🦊/i);
+  expect(foxEmoji).toBeInTheDocument();
+
+  const raccoonEmoji = screen.getByText(/🦝/i);
+  expect(raccoonEmoji).toBeInTheDocument();
+});
```

Comment:
This is a good start to testing, but we want to be testing user interactions. So, for example, we could look for the rat to NOT be in the document and then click the rat button and then expect the rat to be in the document.